src/software/ai/hl/stp/tactic/dribble/dribble_tactic_test.cpp

```diff
@@ -185,7 +185,7 @@ TEST_F(DribbleTacticTest, test_moving_ball_dribble_dest_and_orientation)
     Point initial_position    = Point(-2, 1.5);
     Point dribble_destination = Point(-1, 2);
     Angle dribble_orientation = Angle::zero();
-    BallState ball_state(Point(2, -2), Vector(1, 2));
+    BallState ball_state(Point(1, 0), Vector(1, 2));
```

Comment:
changed because the ball was stopping in the enemy defense area, which the robot cant enter